infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/KubernetesInfraModule.java

```diff
@@ -109,7 +109,7 @@ protected void configure() {
     namespaceConfigurators.addBinding().to(WorkspaceServiceAccountConfigurator.class);
     namespaceConfigurators.addBinding().to(UserProfileConfigurator.class);
     namespaceConfigurators.addBinding().to(UserPreferencesConfigurator.class);
-    namespaceConfigurators.addBinding().to(GitconfigUserDataConfigurator.class);
+    namespaceConfigurators.addBinding().to(GitconfigConfigmapConfigurator.class);
```

Comment:
Can we just call it `GitconfigConfigurator` ?

Reply:
done